packages/entityanalytics_ad/changelog.yml

```
@@ -1,4 +1,9 @@
 # newer versions go on top
+- version: "0.10.0"
+  changes:
+    - description: Changes the field used to populate `user.name` to `sam_account_name`.
+      type: bugfix
+      link: https://github.com/elastic/integrations/pull/13084
```

```suggestion
- version: "0.10.0"
  changes:
    - description: Changes the field used to populate `user.name` to `sam_account_name`.
      type: enhancement
      link: https://github.com/elastic/integrations/pull/13084
```

Sorry if I was not clear earlier 😄 . This is what I meant.